### PR TITLE
Correct SDKObjectAlreadyExistError class and alert message when fcp_template exists

### PR DESCRIPTION
**Pull request by renpei123**
This PR is used to fix below issues.
1. Correct SDKObjectAlreadyExistError class to fix TypeError issue.
2. Correct alert message when fcp_template exists, remove 'already exist' as the super method has this text.

### CODE REVIEW

File: zvmsdk/database.py [hunk 1]
```diff
@@ -1159,8 +1159,8 @@ def create_fcp_template(self, fcp_template_id, name, description,
             # if already exist, raise exception
             if self.fcp_template_exist_in_db(fcp_template_id):
                 raise exception.SDKObjectAlreadyExistError(
-                        obj_desc=("FCP template '%s' already "
-                                  "exist" % fcp_template_id),
+                        obj_desc=("FCP template '%s' "
```
Comment (jackydalong) [inline zvmsdk/database.py:1162]: Replace `"FCP template '%s'  "` to `"FCP device template (id: %s) "`

**Review by jackydalong — CHANGES_REQUESTED** [review]
Left comment
@renpei123

File: zvmsdk/database.py (hunk 1 above)
Comment (renpei123) [reply zvmsdk/database.py]: Replaced.

**Comment by haolp** [comment]
@renpei123 when you perform git commit, can add '-s' which will sign off with your signature.

**Review by jackydalong — APPROVED** [review]
LGTM

**Comment by renpei123** [comment]
Correct alert message when fcp_template exists issue has been done in PR: https://github.com/openmainframeproject/feilong/pull/664.
Close current PR and raise PR (https://github.com/openmainframeproject/feilong/pull/666) to 
correct SDKObjectAlreadyExistError class only.

**Pull request closed without merging**